### PR TITLE
Fix #260 Add select all/clear all checkbox for books panel

### LINKED ISSUE #260: Add select/deselect button for books panel
It would be nice to be able to select/deselect all chapters across all books. This would be done in the books panel like it already is in the chapters panel. De-selecting books would also clear all selected chapters:
Book select all

### PR DESCRIPTION
**Pull request by JCFarrow**
Changed the labels from "Un-select all" to "Clear all" as recommended by the Microsoft Writing Style Guide:
https://docs.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/u/uncheck-unmark-unselect

Changed the way the select all checkboxes work when some are already selected, from selecting all to clearing all as this is how it works in gmail. I tried finding some official recommendation as to how this should work but couldn't find anything.

### CODE REVIEW

File: src/App/store/AppState.ts [hunk 1]
```diff
@@ -194,6 +194,30 @@ export class Project implements BKProject {
...
+      return _.every(b.chapters, 'isSelected');
+    });
+  }
+
+  @action.bound
+  toggleAllChapters(): void {
+    const isSelected = this.isSelected;
+    this.books.forEach((b: Book) => {
+      b.chapters.forEach((c: Chapter) => {
+        c.isSelected = !isSelected;
```
Comment (irahopkinson) [inline src/App/store/AppState.ts:216]: Is there some reason this doesn't call `c. toggleAllChapters()`?

Comment (JCFarrow) [reply src/App/store/AppState.ts]: Ok, my first thought when I saw this was that using toggleAllChapters would have been a much better way. But it turns out that the keyword is "toggle". It toggles all the chapters within each book. So if I have some books selected and some not selected then pushing the clear all button will make the unselected ones selected and clear the selected ones. "Toggling" them all.

But I did clean up a couple of the methods above by reusing the book methods.

**Review by irahopkinson — APPROVED** [review]
👍 LGTM

File: src/App/store/AppState.ts [hunk 2]
```diff
@@ -194,6 +194,30 @@ export class Project implements BKProject {
     );
   }
 
+  @computed({ keepAlive: true })
+  get isSelected(): boolean {
+    return this.books.some((book: Book) => {
+      return book.isSelected;
```
Comment (irahopkinson) [inline src/App/store/AppState.ts:200]: nit: there is a simpler form for this return, i.e. no `{}` and no `return`. Here and below.

**Pull request merged**